### PR TITLE
DOCS-4920-Added included and excluded components

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Comment by paigehargrave** [comment]
@federicoboerr - What does 'online version' mean? Also, do we need any information in https://docs.google.com/spreadsheets/d/1POFJzrB_B2Vn920WY6KwR38XOeyxOuUhSWL6qIa8Kio/edit?ts=5d3adfc2&actionButton=1#gid=816241206 - this might be too detailed but want your opinion. Thanks in advance!

**Comment by federicoboerr** [comment]
`online version` == `cloudhub`== MuleSoft's ran and maintained `Anypoint Platform` in the cloud.

**Comment by federicoboerr** [comment]
We need the detail on which features were added since the previous version.
This is because the cloud keeps adding features and customers expect to see them in PCE and have questions about what's included in PCE and what not.
`Runtime Manager` in the cloud might include features that are not in this version of PCE.
That's why we wanted the list of features that are added in the new versions.

**Comment by paigehargrave** [comment]
@federicoboerr Added the feature table - what is `CS`? Core Services?

File: modules/ROOT/pages/upgrade.adoc [hunk 1]
```diff
@@ -13,6 +13,58 @@ This topic provides information about upgrading Anypoint Platform Private Cloud
 | 1.5.x | 1.6.1, 1.7.1 | 2.0 | Must upgrade to 1.6.1, then 1.7.1 before upgrading to 2.0.
 |===
 
+The following table summarizes the features included in Anypoint Platform Private Cloud Edition version 1.7 and version Anypoint Platform Private Cloud Edition 2.0.
+
+[%header%autowidth.spread]
```
Comment (federicoboerr) [inline modules/ROOT/pages/upgrade.adoc]: update with the changes in the components.

**Comment by federicoboerr** [comment]
`CS` = `Core Services` = `Access Management`
I'd use `Access Management`

File: modules/ROOT/pages/upgrade.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc]: Got it - thank you!

**Comment by paigehargrave** [comment]
@federicoboerr PTAL at latest updates. Thank you!

**Comment by federicoboerr** [comment]
@paigehargrave is the upgrade doc the best place to put the new feature list?

**Comment by paigehargrave** [comment]
@federicoboerr I'll move it - I put it there when I thought we were comparing 1.7 to 2.0. :) Stay tuned.

**Comment by paigehargrave** [comment]
@federicoboerr Ready for your review. Thank you!

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -3,26 +3,62 @@ ifndef::env-site,env-github[]
...
 
-This topic describes the Anypoint Platform Private Cloud Edition. The Anypoint Platform Private Cloud Edition provides the following application components available in the online version of Anypoint Platform:
-
-* Runtime Manager: enables you to track and manage your deployed Mule applications
-* API Manager: enables you to manage your registered APIs
-* Anypoint Exchange: enables your organization to share various assets that can be reused across integration projects.
-* Access Management
-* Design Center, excluding Flow Designer
-
-The following components are not included in the online version of Anypoint Platform:
```
Comment (federicoboerr) [inline modules/ROOT/pages/index.adoc]: Where did this go?

Comment (paigehargrave) [reply modules/ROOT/pages/index.adoc]: OK - added the info back in - wasn't sure how much detail was needed.

**Review by federicoboerr — APPROVED** [review]
LGTM

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -5,6 +5,14 @@ endif::[]
 
 This topic describes the Anypoint Platform Private Cloud Edition. Anypoint Platform Private Cloud Edition enables you to run and manage Mule applications on your local servers, using the security policies your system requires. When running Anypoint Platform Private Cloud Edition, MuleSoft systems do not interact with your data. You configure and run the software and maintain all storage, processing and transmission of data.
 
+The Anypoint Platform Private Cloud Edition provides the following application components when combined with CloudHub:
+
+* Runtime Manager: enables you to track and manage your deployed Mule applications
```
Comment (hannanelson) [inline modules/ROOT/pages/index.adoc]: Kim likes a newline after the :

Comment (hannanelson) [reply modules/ROOT/pages/index.adoc]: Suggest no : but make each of these full sentences with terminating punctuation.
Runtime Manager enables you to track and manage your deployed Mule applications.

File: modules/ROOT/pages/upgrade.adoc [hunk 4]
```diff
@@ -20,38 +20,38 @@ The following table summarizes the new features included in Anypoint Platform Pr
...
+| Exchange |	Automatically generate rich documentation for RAML and OAS specifications.
+| Exchange |	Perform interactive API use case validation through API Notebook.
+| Exchange |	Use a new mocking service for testing API calls without API implementation.
+| Exchange |	Register clients and request access for APIs managed by Anypoint Platform.
+| Exchange |	Automatically index API instances and endpoints via Exchange.
+| Exchange |	Share APIs externally using Exchange portal.
+| Exchange |	Customize and brand features of Exchange portal.
+| Exchange |	Leverage HTML support for content inserted in a Markdown editor (limitations apply).
+| Exchange |	Save a search for organizations that provides consistent search navigation to all business group users.
+| Exchange |	Utilize personalized saved searches to save frequently used searches.
```
Comment (hannanelson) [inline modules/ROOT/pages/upgrade.adoc]: Use?

**Review by hannanelson — APPROVED** [review]
Some wee changes (take or leave).

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/index.adoc]: Thank you - I updated the wording.

File: modules/ROOT/pages/upgrade.adoc (hunk 4 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc]: Updated.

**Comment by paigehargrave** [comment]
@hannanelson - Thank you!

**Pull request merged**